cannon/cannon.py

```diff
@@ -0,0 +1,190 @@
+# Author: Sathish Kumar E
+
+import pygame
+import sys
+import os
+from random import randint
+from pygame.locals import *
+from math import sin, cos, pi
+
+WIN_WIDTH = 600
+WIN_HEIGHT = 600
+FPS = 30
+pygame.font.init()
+pygame.init()
+STAT_FONT = pygame.font.SysFont("comicsans", 30)
+
+
+def rotate(surface, angle, pivot, offset):
+    """Rotate the surface around the pivot point.
+
+    Args:
+                    surface (pygame.Surface): The surface that is to be rotated.
+                    angle (float): Rotate by this angle.
+                    pivot (tuple, list, pygame.math.Vector2): The pivot point.
+                    offset (pygame.math.Vector2): This vector is added to the pivot.
+    """
+    rotated_image = pygame.transform.rotozoom(
+        surface, -angle, 1)  # Rotate the image.
+    rotated_offset = offset.rotate(angle)  # Rotate the offset vector.
+    # Add the offset vector to the center/pivot point to shift the rect.
+    rect = rotated_image.get_rect(center=pivot+rotated_offset)
+    return rotated_image, rect  # Return the rotated image and shifted rect.
+
+
+class Plank:
+    PLANK_WIDTH = 150
+    PLANK_HEIGHT = 30
+
+    def __init__(self):
+        self.xvel = 1
+        self.rect = pygame.Rect(0, 0, self.PLANK_WIDTH, self.PLANK_HEIGHT)
+        self.rect.center = (300, 550)
```

Comment:
One more suggestion is, that you have hardcoded these values. I will recommend you to create all these values(coordinate, etc) taking window size in reference. So that in future if we change window size than other size get adjusted according to that.

For example: 
```
paddle_width = window_width/3
```

Reply:
You can look at other env for reference.